std/file.d

```diff
@@ -886,11 +886,20 @@ version (Windows) private void writeImpl(scope const(char)[] name, scope const(F
 }
 
 /***************************************************
- * Rename file `from` _to `to`.
+ * Rename file `from` _to `to`, moving it between directories if required.
  * If the target file exists, it is overwritten.
+ *
+ * On Posix, the operation is atomic, which means, that it is not possible to
+ * rename a file across different mount points. If `to` already exists, it
```

Comment:
AFAIK it's not possible to rename a file across volumes on Windows either. Generally this means across drives, but with the introduction of reparse points, any two paths may conceivably fail, as on POSIX.

Reply:
`rename` is part of C, and is specified to be atomic in the standard, so probably none of this is actually specific to POSIX other than the "mount points" term.

Reply:
I have little experience with Windows. Changed this now.